### PR TITLE
Fixes #1520 - Add ariaLabel support and remove not descriptive 'toggle'

### LINKED ISSUE #1520: ClayDropdownBase aria-label are static and not descriptive
The aria-label is always "toggle" independently of action and language

### PR DESCRIPTION
**Pull request by boton**
https://issues.liferay.com/browse/IFI-464

### CODE REVIEW

File: packages/clay-dropdown/src/ClayDropdownBase.soy [hunk 1]
```diff
@@ -9,6 +9,7 @@
 	{@param? _handleButtonClick: any}
 	{@param? _handleItemClick: any}
 	{@param? _handleSearch: any}
+	{@param? ariaLabel: string}
```
Comment (matuzalemsteles) [inline packages/clay-dropdown/src/ClayDropdownBase.soy]: hey @boton, can you declare this parameter in the state of `ClayDropdownBase`?

File: packages/clay-dropdown/src/ClayActionsDropdown.soy [hunk 2]
```diff
@@ -8,6 +8,7 @@
 	{@param spritemap: string}
 	{@param? _handleButtonClick: any}
 	{@param? _handleItemClick: any}
+	{@param? ariaLabel: string}
```
Comment (matuzalemsteles) [inline packages/clay-dropdown/src/ClayActionsDropdown.soy]: hey @boton, can you declare this parameter in the state of `ClayActionsDropdown`?

**Comment by jbalsas** [comment]
Hey @matuzalemsteles, @boton will be on vacation for a couple of weeks. Could you take care of cleaning this up?

Thanks! 👍

**Comment by matuzalemsteles** [comment]
yes, thanks for the info.

**Comment by matuzalemsteles** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Comment by matuzalemsteles** [comment]
Resent #1529

**Pull request closed without merging**